tests/test_api.py

```diff
@@ -1281,6 +1285,58 @@ def test_submit_manifest_table_and_file_upsert(
         )
         assert response_csv.status_code == 200
 
+    @pytest.mark.synapse_credentials_needed
+    @pytest.mark.submission
+    def test_submit_nested_manifest_table_and_file_upsert(
```

Comment:
`upsert` in schematic means appending rows in an existing table, which is not what this function is doing. Could you remove the word "upsert" here in the function?

Reply:
Good catch! I updated this